### ISSUE TITLE
where is the loss function of the sequence labeling?

### ISSUE THREAD

**Issue by jbjeong91** [WindChimeRan/OpenJERE#3]

**Comment by jbjeong91** [comment]
> ## Hello,
> I can't find the loss of the sequence labeling in your code.
> I just can find the loss for the decoder as follows:
> ```
>     loss = 0
>     for t in range(self.seq2seq.decoder.decodelen):
>         loss = loss + self.loss(pred_logits_list[t], triplets[:, t])
> ```
